lib/transport_web/controllers/discussions_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule TransportWeb.DiscussionsController do
```

Comment:
What about ```Conversation``` rather than ```Discussion```?

Reply:
It would differ from datagouvfr vocabulary.